### PR TITLE
Revise details of JS chat guide basic

### PR DESCRIPTION
**Pull request by chpapa**
@tensiuyan I have merged your revision to JS chat guide basic, but I think there are something we should change, can you revise the changes and let me know if it is good to merge?

### CODE REVIEW

File: chat/basics/js.md [hunk 1]
```diff
@@ -30,6 +29,8 @@ And here are a list of Sample Projects using Skygear JS Chat SDK:
   Conversation; although they are included as Example.
 * We should also explain conversation options are configurable via
   [`updateConversation`](https://doc.esdoc.org/github.com/skygeario/chat-SDK-JS/class/lib/container.js~SkygearChatContainer.html#instance-method-updateConversation)
+* We should explain or include a link to the UIKit for quick chat application
```
Comment (tensiuyan) [inline chat/basics/js.md:32]: but do we have a UIkit for web now? or do you mean for future?

**Review by tensiuyan — COMMENTED** [review]
Ben I'm good with the changes. Only for the UIkit section. Just wanted to know if we already have a UIkit for JS or it is just jotted down for future reference?

File: chat/basics/js.md (hunk 1 above)
Comment (chpapa) [reply chat/basics/js.md:32]: Future XD

**Comment by tensiuyan** [comment]
Ben - should I merge this pull request?

**Pull request merged**